yarn-project/aztec.js/src/wallet/account_manager.ts

```diff
@@ -81,7 +89,7 @@ export class AccountManager {
    * @returns An entrypoint.
    */
   public async getAccountInterface(): Promise<AccountInterface> {
-    const nodeInfo = await this.pxe.getNodeInfo();
+    const nodeInfo = await this.node.getNodeInfo();
```

Comment:
Wallet now provides `getChainInfo` (which I believe it's what the account interface wants) so this file might not need changes

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/17109/commits/7a405c36b514430e8249b1c59957c7642887debc